openai.py

```diff
@@ -2,6 +2,7 @@
 import sys

```

Comment:
There are critical issues that need attention regarding input validation and indentation errors. Ensure environment variables are properly validated, and fix the incorrect indentation in the 'post_review' function.

```python
try:
    response = requests.post(url, headers=headers, json=review_data)
    response.raise_for_status()
    print("Review posted successfully.")
except requests.exceptions.RequestException as e:
    print(f"Failed to post review: {e}")
    print(f"Response content: {response.content}")
```